sdk/core/System.ClientModel/src/NetStandardPolyfill/TrimmingAttribute.cs

```diff
@@ -1,8 +1,6 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#nullable enable
```

Comment:
Why was this removed?

Reply:
Nullable is enabled for the whole System.ClientModel library so we don't need it the inline directives.